packages/grafana-ui/src/components/VizLayout/VizLayout.tsx

```diff
@@ -46,7 +47,11 @@ export const VizLayout: VizLayoutComponentType = ({ width, height, legend, child
     );
   }
 
-  const { placement, maxHeight = '35%', maxWidth = '60%' } = legend.props;
+  let { placement, maxHeight = '35%', maxWidth = '60%' } = legend.props;
```

Comment:
I had to return `let` here. Initially, I did not realized why I have changed it so I put `const` back which doesn't work obviously, since `placement` is overwritten bellow. So keeping `let`.